pom.xml

```
@@ -119,13 +119,77 @@
                                 <version>8.29</version>
                             </dependency>
                         </dependencies>
-                        <configuration>
+                        <configuration>     
                             <configLocation>strongbox-checks.xml</configLocation>
                             <consoleOutput>true</consoleOutput>
                         </configuration>
                     </plugin>
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>Enforce-Java-11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <!--<version>3.0.0-M3</version>-->
+                        <executions>
+                            <execution>
+                                <id>enforce-versions</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireJavaVersion>
+                                            <version>[11.3,)</version>
+                                            <message>Strongbox only supports JDK 11.3 and up.</message>
+                                            <level>ERROR</level>
+                                        </requireJavaVersion>
+                                    </rules>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>Enforce-Java-8</id>
```

Please, change this to:
```suggestion
            <id>enforce-jdk8</id>
```

---

resolved